python/GafferSceneTest/FilterResultsTest.py

```diff
@@ -183,5 +183,29 @@ def testComputeCacheRecursion( self ) :
 		Gaffer.ValuePlug.clearCache()
 		script["filterResults2"]["out"].getValue( h )
 
+	@GafferTest.TestRunner.PerformanceTestMethod()
+	def testHashPerf( self ):
+
+		name = "longName" * 50
```

Comment:
Using a Duplicate as a cheap way of making lots of locations is fine (we do have scenes with lots of locations, and for this test we want to make them cheaply so we're testing the FilterResults). But this artificially long name is a bit iffy - it's completely unrepresentative. If I remove the `* 50` I do see slightly less impressive results ( 64% on 32 cores, 20% on 8), but they're still great results, and I'd rather that than slightly-more-great-but-less-representative.